pkg/controller/certificates/webhook_signer_config.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package certificates
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/golang/glog"
+
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+const (
+	defaultRetryBackoff = time.Duration(500) * time.Millisecond
+	useDefault          = time.Duration(0) // missing/zero value indicates to use default
+)
+
+type webhookSignerConfig struct {
```

Comment:
D'oh. My biggest weakness: TODO comments mere lines away from where I stopped reading :-P.

I like the idea of removing the need for yet another config file anyway, especially since I doubt many users will care enough to override the `retryBackoff`. Will do.